.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Generate changelog and plugin archive for new release
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
+        run: ./gradlew buildPlugin
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/}
+      - name: Simple conventional changelog
+        uses: lstocchi/simple-conventional-changelog@0.0.6
+        id: changelog
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          current-tag: ${{ steps.get_version.outputs.VERSION }}
+          types-mapping: 'feat:Features,fix:Bug Fixes,docs:Documentation,refactor:Refactoring,build:Builds,chore:Other'
+          scopes-mapping: 'UI:UI,Hub:Hub'
+      - run: |
+          echo '${{ steps.changelog.outputs.changelog }}'
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.get_version.outputs.VERSION }}
+          release_name: ${{ steps.get_version.outputs.VERSION }}
+          body: ${{ steps.changelog.outputs.changelog }}
+      - name: Attach zip to release
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: '${{ github.workspace }}/build/distributions/intellij-common-${{ steps.get_version.outputs.VERSION }}.zip'
+          asset_name: 'intellij-${{ steps.get_version.outputs.VERSION }}.zip'
```

Comment:
Is this wanted? `intellij-version` instead of `intellij-common-version`

Reply:
Good catch. Fixed now